src/pyfaf/opsys/centos.py

```diff
@@ -187,7 +193,20 @@ def save_ureport(self, db, db_report, ureport, packages, flush=False, count=1):
             db.session.flush()
 
     def get_releases(self):
-        return {"7": {"status": "ACTIVE"}}
+        result = {}
+
+        for release in re.split(r"\s*[,;]\s*|\s+", self.inactive_releases):
+            result[release] = {
+                "status": "EOL",
+                "shortname": "centos-{0}".format(release)
+            }
+        for release in re.split(r"\s*[,;]\s*|\s+", self.active_releases):
+            result[release] = {
+                "status": "ACTIVE",
+                "shortname": "centos-{0}".format(release)
```

Comment:
why the shortname? it was not previously in that dictionary.

Reply:
I suppose i just wanted to keep it consistent with the [way its done for Fedora](https://github.com/abrt/faf/blob/f18b566afa2abfbd61d9278a44b6488b36d10430/src/pyfaf/opsys/fedora.py#L229), but you're right that it doesn't seem to be needed for either of the systems at the moment.

Reply:
I think it’s a derelict from the PkgDB days, but even then it’s not quite clear what it was for.

Reply:
I see. Should I remove it from the Fedora part as well while I'm at it?

Reply:
I can’t see any harm coming from doing it, so why not.